__tests__/app/components/playList.test.tsx

```diff
@@ -0,0 +1,57 @@
+import PlayList from '../../../app/components/playList';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import IPlaylist from '@/app/playlist/iPlaylist';
+import { faker } from "@faker-js/faker";
+import { getPlayLists } from '../../../app/lib/api-service';
+
+jest.mock('../../../app/lib/api-service', () => ({
+    getPlayLists: jest.fn()
+}));
+
+jest.mock("../../../app/components/gridAction", () => () => {
+    return <td />;
+});
+
+describe('playlist', () => {
+    afterAll(() => {
+        jest.clearAllMocks();
+    });
+
+    const playLists: Array<IPlaylist> = [{ 
+        id: faker.string.alphanumeric(),
```

Comment:
We are mocking same level of data in all the files. Create a fixture class eg : playlist.ts and add the common data and fetch from that class

Reply:
added fixture & got data from it.